src/Data/Fin/Properties.agda

```diff
@@ -105,9 +105,27 @@ toℕ-strengthen : ∀ {n} (i : Fin n) → toℕ (strengthen i) ≡ toℕ i
 toℕ-strengthen zero    = refl
 toℕ-strengthen (suc i) = cong suc (toℕ-strengthen i)
 
-toℕ-raise : ∀ {m} n (i : Fin m) → toℕ (raise n i) ≡ n ℕ.+ toℕ i
+------------------------------------------------------------------------
+-- ↑ˡ
+------------------------------------------------------------------------
+
+_toℕ-↑ˡ_ : ∀ {m} (i : Fin m) n → toℕ (i ↑ˡ n) ≡ toℕ i
```

Comment:
I know this is divisive, but we don't usually have infix _proofs_ in the standard library.

Reply:
Oh, OK. Not intended to be divisive at all. I find it helps me remember the argument order if proof-follows-function in arg position and fixity. But that's cool.